src/python/WMCore/WorkQueue/WorkQueue.py

```diff
@@ -413,6 +413,16 @@ def _getDBSBlock(self, match, wmspec):
                                            splitedBlockName['Offset'],
                                            splitedBlockName['NumOfFiles'])
 
+            # If TrustSitelist is enabled, then assume files without location are
+            # available at the ACDC fileset location
+            if match['NoInputUpdate']:
+                acdcLocations = match['Inputs'].values()[0]
```

Comment:
Where we get the fileset location?

Reply:
I think I meant ACDC collection location. Which is the location from the workqueue element.